data/PriyanshShrivastava.json

```diff
@@ -0,0 +1,33 @@
+{
+	"name": "Priyansh Shrivastava",
+	"displayStatsPublic": true,
+	"type": "personal",
+	"bio": "Full Stack Developer | DevOps | Javascript | Node.js | TailwindCSS ",
+	"avatar": "https://github.com/PriyanshShrivastava.png",
```

Comment:
```suggestion
```
`avatar` is no longer needed. Picture will be fetched automatically from your GitHub account